icu4c/source/data/misc/units.txt

```diff
@@ -1140,21 +1135,6 @@ units:table(nofallback){
                         unit{"inch"}
                     }
                 }
-                CN{
-                    {
-                        unit{"meter-and-centimeter"}
-                    }
-                }
-                DE{
-                    {
-                        unit{"meter-and-centimeter"}
```

Comment:
Thanks for dealing with the unit test!

(I want synthetic testing data in ICU unit tests!)